pom.xml

```diff
@@ -61,6 +61,11 @@
       <version>1.3</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+      <artifactId>icon-shim</artifactId>
```

Comment:
Looking at the removed jelly code it seems like the correct core dependency is already there so you shouldn't need this plugin.